From 48acbc87a83a3d1f34f3b2383d42fcf02f7568bd Mon Sep 17 00:00:00 2001
From: Tomas Korbar <tkorbar@redhat.com>
Date: Mon, 14 Jan 2019 16:07:25 +0100
Subject: [PATCH 10/11] Fix writing of advanced settings

After reading from stream we must return again to line 0 if we
want to read file again
---
 serversettings.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/serversettings.py b/serversettings.py
index c98b08b1..72423057 100644
--- a/serversettings.py
+++ b/serversettings.py
@@ -440,6 +440,9 @@ class ServerSettings(GtkGUI):
                 has_browsepoll = True
                 break
 
+        # Return to the start of file
+        f.seek(0)
+
         for line in f:
             line = line.decode('UTF-8')
             l = line.lower ().strip ()
-- 
2.20.1

